### ISSUE TITLE
crash when adding new task

### ISSUE THREAD

**Issue by aleksandrpro** [dmfs/opentasks#645]
I have Sony XA with Android 7.0.
After tapping "save" button app crashes and new task does not save.
No matter which task list i use: local or davdroid synced.

adb logcat says:
```
01-23 11:39:32.857 25962 25962 E AndroidRuntime: FATAL EXCEPTION: main
01-23 11:39:32.857 25962 25962 E AndroidRuntime: Process: org.dmfs.tasks, PID: 25962
01-23 11:39:32.857 25962 25962 E AndroidRuntime: java.lang.NumberFormatException: For input string: "null"
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at java.lang.Long.parseLong(Long.java:443)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at java.lang.Long.parseLong(Long.java:485)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at org.dmfs.tasks.utils.RecentlyUsedLists.getList(RecentlyUsedLists.java:51)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at org.dmfs.tasks.utils.RecentlyUsedLists.use(RecentlyUsedLists.java:108)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at org.dmfs.tasks.EditTaskFragment.saveAndExit(EditTaskFragment.java:786)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at org.dmfs.tasks.EditTaskFragment.onOptionsItemSelected(EditTaskFragment.java:539)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v4.app.Fragment.performOptionsItemSelected(Fragment.java:2379)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v4.app.FragmentManagerImpl.dispatchOptionsItemSelected(FragmentManager.java:3133)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v4.app.FragmentController.dispatchOptionsItemSelected(FragmentController.java:344)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v4.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:414)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:195)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:113)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.app.AppCompatDelegateImplV9.onMenuItemSelected(AppCompatDelegateImplV9.java:679)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:822)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:156)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:969)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:959)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.widget.ActionMenuView.invokeItem(ActionMenuView.java:623)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.support.v7.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:154)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.view.View.performClick(View.java:5723)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.view.View$PerformClick.run(View.java:22689)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:836)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:103)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:203)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6361)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1063)
01-23 11:39:32.857 25962 25962 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:924)
01-23 11:39:32.864  4885 30518 W ActivityManager:   Force finishing activity org.dmfs.tasks/.EditTaskActivity

```

**Comment by lemonboston** [comment]
Thanks for the report, this looks like a duplicate of https://github.com/dmfs/opentasks/issues/562#issuecomment-351180491 and should be fixed in the next release.

**Comment by Ganzal** [comment]
Same crashes on Samsung A7 2017 (sm-a720f), Android 8.0.0

**Comment by aleksandrpro** [comment]
New version still not released. App reinstall resolved my issue.


пн, 4 июн. 2018 г., 22:41 Sergey D. Ivanov <notifications@github.com>:

**Comment by Ganzal** [comment]
I can confirm that reinstalling did the trick, but some time later issue comes back again.

**Comment by dmfs** [comment]
fixed in 1.1.14
